doc/read-the-docs-site/background/credentials.rst

```diff
@@ -93,14 +105,17 @@ Script credentials offer many benefits over public key credentials:
 * Via the datum, they can be associated with supplemental information, which
   can be used for identity verification.
 * They can have failsafe options built into them to allow recovery of the credential
-  if auxillary data such as private keys are compromised.
+  if auxillary data, such as private keys, are compromised.
+
+Disadvangates of script credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 However, they also have several disadvantages:
 
-* They are complicated and expensive to create and used compared with public
```

Comment:
Suggest slightly revised version:

* They are complicated and expensive to create and use compared with public key credentials.
* If not carefully written and tested, they can have unintended edge cases or bugs that compromise security.
* They consume more space in transactions and are more expensive to validate due to the script bytes, datum, and redeemer. This increases transaction fees.
* They require significant upfront investment, especially for high-risk applications, and are costly to operate, necessitating higher transaction fees and technical expertise, either from trained staff or commercial service providers who may charge additional fees.